libc/utils/gpu/loader/CMakeLists.txt

```diff
@@ -56,11 +37,10 @@ elseif(TARGET nvptx-loader AND LIBC_TARGET_ARCHITECTURE_IS_NVPTX)
   )
 endif()
 
-if(TARGET libc.utils.gpu.loader)
-  get_target_property(gpu_loader_tgt libc.utils.gpu.loader "TARGET")
+foreach(gpu_loader_tgt amdhsa-loader nvptx-loader)
   if(gpu_loader_tgt)
```

Comment:
Are they always available to install now?  If so remove the `if` statement.

Reply:
No, they can only be built if the user had HSA or CUDA installed on their machine in a location it could find.

Reply:
So, we have a lot of other tools that depend on CUDA / HSA that are installed unconditionally (see amdgpu-arch) but those use `dlopen`. I could do the same for my loader utilities but that's a lot of work for what is essentially a test tool.

Reply:
I mean `${gpu_loader_tgt}` is either `"amdhsa-loader"` or `"nvptx-loader"`, so the check `if(gpu_loader_tgt)` is always true, isn't it?